azure-pipelines.yaml

```diff
@@ -42,6 +44,16 @@ jobs:
   - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
     displayName: 'Component Detection'
 
+  - script: dotnet pack /p:PackageVersion=$(Major).$(Minor).$(Patch) -o '$(Build.ArtifactStagingDirectory)' --include-symbols src/Microsoft.Azure.WebJobs.Extensions.Kafka/Microsoft.Azure.WebJobs.Extensions.Kafka.csproj
```

Comment:
Hi @TsuyoshiUshio I would recommend moving this part towards the end of the build process. Right now it is before the end to end tests. It should only happen if test have passed.

Reply:
Nice catch. Agreed.

Reply:
Thanks. :) Done.